easybuild/easyconfigs/p/Python/Python-2.7.13-foss-2016b.eb

```diff
@@ -0,0 +1,134 @@
+name = 'Python'
+version = '2.7.13'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.13.0'),
+    ('Tk', '8.6.5'),  # this requires a full X11 stack
```

Comment:
@verdurin what about the plan to drop the `Tk` dep here, disable `TkInter` in the `Python` module and flesh it out as a stand-alone module instead?

This seems like a good time to tackle that?